docs/01-app/01-getting-started/04-linking-and-navigating.mdx

```diff
@@ -0,0 +1,405 @@
+---
+title: Linking and Navigating
+description: Learn how prefetching, prerendering, and client-side navigation works in Next.js, and how to use the Link Component.
+related:
+  links:
+    - app/api-reference/components/link
+    - app/api-reference/file-conventions/loading
+---
+
+In Next.js, routes are rendered on the server by default. This often means the client has to wait for a server response before a new route can be shown. Next.js comes built-in [prefetching](#prefetching), [streaming](#streaming), and [client-side transitions](#client-side-transitions) ensuring navigation stays fast and responsive.
+
+This guide explains how navigation works in Next.js and how you can optimize it for [dynamic routes](#dynamic-routes-without-loadingtsx) and [slow networks](#slow-networks).
+
+## How navigation works
+
+To understand how navigation works in Next.js, it helps to be familiar with the following concepts:
+
+- [Server rendering](#server-rendering)
+- [Prefetching](#prefetching)
+- [Streaming](#streaming)
+- [Client-side transitions](#client-side-transitions)
+
+### Server Rendering
+
+In Next.js, [Layouts and Pages](/docs/app/getting-started/layouts-and-pages) are [React Server Components](https://react.dev/reference/rsc/server-components) by default. On initial and subsequent navigations, the [Server Component Payload](/docs/app/getting-started/server-and-client-components#how-do-server-and-client-components-work-in-nextjs) is generated on the server before being sent to the client.
+
+There are two types of server rendering, based on _when_ it happens:
+
+- **Static Rendering (or Prerendering)** happens at build time or during [revalidation](/docs/app/getting-started/caching-and-revalidating) and the result is cached.
+- **Dynamic Rendering** happens at request time in response to a client request.
+
+The trade-off of server rendering is that the client must wait for the server to respond before the new route can be shown. Next.js addresses this delay by [pretching](#prefetching) routes the user is likely to visit and performing [client-side transitions](#client-side-transitions).
+
+> **Good to know**: HTML is also generated for the initial visit.
+
+### Prefetching
+
+Prefetching is the process of loading a route in the background before the user navigates to it. By the time the user clicks, the request for the route has already been initiated and the response stored in the browser's cache, making the transition instant.
```

Comment:
`stored in the browser's cache,`

I think in this case, the response, for example rsc, is stored into the client router cache rather, right? 

Maybe:

```suggestion
Prefetching is the process of loading a route in the background before the user navigates to it. This makes navigation between routes in your application feel instant, because by the time a user clicks on a link, the data to render the next route is already available client side. 
```